src/library.js

```diff
@@ -271,12 +271,22 @@ LibraryManager.library = {
   __execvpe: 'execl',
   fexecve: 'execl',
 
-  _exit: function(status) {
+  exit: function(status) {
+#if MINIMAL_RUNTIME
+    throw 'exit('+status+')';
```

Comment:
clearer to read with spaces around the `+`, and that's more consistent with other code (minifier will minify that anyhow)